### DISCUSSION TITLE
How to enable mix with audio with BetterPlayerListVideoPlayerController?

### DISCUSSION THREAD

**Discussion by thegrxp** [jhomlala/betterplayer#573] [Q&A]
It's possible to enable it with a `BetterPlayerController` using `betterPlayerController.setMixWithOthers(true)` but I can't find a way to make it work using a `BetterPlayerListVideoPlayerController`

Thank you!

**Comment by thegrxp** [comment]
Add a function to `BetterPlayerListVideoPlayerController` might be the best? @jhomlala 

```
  void setMixWithOthers(bool mixWithOthers) {
    _betterPlayerController?.setMixWithOthers(mixWithOthers);
  }
```

**Reply by jhomlala** [reply]
@thegrxp This will be added in next release (0.7.3)
